src/main/webapp/app/admin/cleanup-service/cleanup-service.component.html

```diff
@@ -0,0 +1,80 @@
+            <thead>
+                <tr>
+                    <th scope="col" jhiTranslate="cleanupService.table.operation"></th>
+                    <th scope="col" jhiTranslate="cleanupService.table.deleteFrom"></th>
+                    <th scope="col" jhiTranslate="cleanupService.table.deleteTo"></th>
+                    <th scope="col" jhiTranslate="cleanupService.table.lastExecuted"></th>
+                    <th scope="col"></th>
+                </tr>
+            </thead>
+            <tbody>
+                @for (operation of cleanupOperations; track operation) {
+                    <tr>
+                        <td>
+                            <span [jhiTranslate]="'cleanupService.operation.' + operation.name"></span>
+                        </td>
+                        <td>
+                            @if (operation.name !== 'deleteOrphans') {
+                                <div class="col-md-12">
+                                    <jhi-date-time-picker
+                                        id="deleteFromDate"
+                                        [labelName]="'cleanupService.table.deleteFrom' | artemisTranslate"
+                                        [hideLabelName]="true"
+                                        [(ngModel)]="operation.deleteFrom"
+                                        (valueChange)="validateDates(operation)"
+                                        [shouldDisplayTimeZoneWarning]="false"
+                                        [error]="!operation.datesValid()"
+                                    />
+                                </div>
+                            }
+                        </td>
+                        <td>
+                            @if (operation.name !== 'deleteOrphans') {
+                                <div class="col-md-12">
+                                    <jhi-date-time-picker
+                                        id="deleteToDate"
+                                        [labelName]="'cleanupService.table.deleteTo' | artemisTranslate"
+                                        [hideLabelName]="true"
+                                        [(ngModel)]="operation.deleteTo"
+                                        (valueChange)="validateDates(operation)"
+                                        [shouldDisplayTimeZoneWarning]="false"
+                                        [error]="!operation.datesValid()"
+                                    />
+                                </div>
+                                @if (!operation.datesValid()) {
+                                    <span class="invalid-feedback" jhiTranslate="cleanupService.error.datesInvalid"></span>
+                                }
+                            }
+                        </td>
+                        <td>
+                            @if (operation.lastExecuted === undefined) {
```

Comment:
```suggestion
                            @if (!operation.lastExecuted) {
```
Maybe even invert the logic to get rid of the negation